### PR TITLE
Create 2020-08-07-do-you-speak-prestashop-july-2020.md

### PR DESCRIPTION
**Pull request by Junebyun**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Publish July Crowdin Monthly Article: 2020-08-07-do-you-speak-prestashop-july-2020.md
| Type?         | Article
| BC breaks?    | 
| Deprecations? | 
| Fixed ticket? | 
| How to test?  |

### CODE REVIEW

**Comment by Junebyun** [comment]
@LouiseBonnard Crowdin Monthly ready for review! :)

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+|-| ---------- | -------- | ----------------
+ 1. | Lee Kung (grmark) | Thai | 16,569
+ 2. | Manuela Silva (manuelarodsilva) | Portuguese | 1,619
+ 3. | Marcin Orzechowski (Martinovy) | Polish | 455
+ 4. | Gerardas (gerardas) | Lithuanian | 265
+ 5. | Stanislav Yordanov (StanProg) | Bulgarian | 126
+ 6. | ftugrul | Turkish | 60
+ 7. | David Gonzalez (davidglezz) | Spanish | 21
+
+Congrats, and welcome to our new top contributors :clap:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
Congrats, and welcome to our new top contributors! :clap:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+At the end of July 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **4 languages**:
+
+| Chinese Traditional | Danish | Dutch | French | Italian |
+| Portuguese, Brazil | Slovak | Slovene | Spanish | Swedish | 
+
+
+## Languages with the best evolution
+
+In comparison with June 2020, the following languages had the best progress thanks to the translation community :sparkles:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
In comparison with June 2020, the following languages had the best progress thanks to the translation community:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+![Best translation progress for July 2020](/assets/images/2020/07/build-crowdin-progress-july20.png)
+
+A huge thank you to all the contributors!
+
+Of course, this is highlighting the languages that made some progress with new translations; but it doesn't mean that the languages that aren't mentioned here aren't active. Indeed, some editing and rewriting could be going on, but the percentage of translation wouldn't be modified (since it's working on strings that are already translated). So let's not forget about the work of other proofreaders! Thanks to you too!
+
+
+## Languages that need (more) proofreaders
+
+A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important we should keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations :fire:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important we should keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations! :fire:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+* **Spanish, Venezuela** (52% vs 11%).
+* **Galician** (100% vs 59%).
+* **Esperanto** (40% vs 0%).
+* **Spanish, Mexico** (93% vs 55%).
+
+![Languages that need proofreading](/assets/images/2020/07/build-crowdin-proofreading-july20.png)
+
+If you wish to help to proofread what has been translated, please contact us with the language you'd like to proofread: just send an email to translation@prestashop.com. We need your help!
+
+If you haven't joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official) :wink:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
If you haven't joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official)! :wink:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+If you want to gather your fellow translators to work towards a better harmonization, start a glossary, or anything else, do let us know: we'll include a word about it in the next monthly report.
+
+Do you have a question, a remark? Don't hesitate to leave a comment. See you next month :raising_hand:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
Do you have a question, a remark? Don't hesitate to leave a comment. See you next month! :raising_hand:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+
+### Interested in testing the TM tool? 
+
+This test was done for the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. Any latin american spanish native who wishes to help and become more involved in projects by becoming proofreaders are more than welcome. 
+
+I also want to let contributors know that if you are interested in testing the TM tool to translate more strings of the language project you are working on, you can [reach out to me](https://crowdin.com/profile/june.byun)! If translations done by the TM tool are proved to be accurate, this method could be replicated and make your work easier and make translations available for merchants - and you as well. :wink:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]: It's interesting, you should consider writing a dedicated blog post instead. ;-)

Comment (LouiseBonnard) [reply news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
### First news, a way to boost translations :fire:

As I mentioned in the [previous Crowdin Monthly](https://build.prestashop.com/news/do-you-speak-prestashop-june-2020/), many PrestaShop translation projects, such as Esperanto, are reaching higher translation rates thanks to dedicated contributors. To help and encourage them, I did some tests by translating strings with Crowdin’s Pre-translation via Translation Memory (TM) tool. What is this tool and how does it work? 

The Project Translation Memory (TM) is the vault of the source strings and their translations into different languages. Every translation made in a project is automatically added to the Project Translation Memory. By using the Pre-translation via the TM tool, translations of the same or similar strings in a project can be applied to untranslated strings. As strings of older versions are propagated to new versions, the more the older versions are translated, the less you will have to translate for new versions. :sparkles:


### Interested in testing the TM tool? 

This test was done for both the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. Any Latin American Spanish native who wishes to help and become more involved in projects by becoming proofreaders are more than welcome. 
```

Comment (LouiseBonnard) [reply news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]: I'm not in favor of the last paragraph: in my opinion, the TM tool should remain an exception in the translation process, and definitely not be an option for any contributor.

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md (hunk 1 above)
Comment (LouiseBonnard) [reply news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]: Pleeeaase, make sure the grammar is flawless before adding emojis.
🙏🙏🙏

**Review by LouiseBonnard — COMMENTED** [review]
Nice, thanks @Junebyun! Note that the images have been uploaded in the July directory while they should be in the August one but that's alright. :-)

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – July 2020 edition"
+subtitle: "Where you'll learn that that there is a way to boost translations"
```
Comment (Progi1984) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]: `that that` ?

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+* **19 members** were active on the project this month.
+* A total of **24,007 words have been translated** and **19,115 validated**.
+* All this in **15 different languages**.
+
+Thank you for your involvement!
+
+
+## Top contributors
+
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and we can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for July 2020.
```
Comment (Progi1984) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for July 2020.
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md (hunk 3 above)
Comment (Progi1984) [reply news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important PrestaShop should keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations! :fire:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+* **Spanish, Argentina** (100% translated vs 13% validated).
+* **Spanish, Venezuela** (52% vs 11%).
+* **Galician** (100% vs 59%).
+* **Esperanto** (40% vs 0%).
+* **Spanish, Mexico** (93% vs 55%).
+
+![Languages that need proofreading](/assets/images/2020/07/build-crowdin-proofreading-july20.png)
+
+If you wish to help to proofread what has been translated, please contact us with the language you'd like to proofread: just send an email to translation@prestashop.com. We need your help!
```
Comment (Progi1984) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
If you wish to help to proofread what has been translated, please contact us with the language you'd like to proofread: just send an email to translation@prestashop.com. The PrestaShop project needs your help!
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md (hunk 4 above)
Comment (Progi1984) [reply news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
If you haven't joined the PrestaShop project on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official)! :wink:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+If you haven't joined us on Crowdin yet, [it's never too late](https://crowdin.com/project/prestashop-official) :wink:
+
+If you want to gather your fellow translators to work towards a better harmonization, start a glossary, or anything else, do let us know: we'll include a word about it in the next monthly report.
```
Comment (Progi1984) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
If you want to gather your fellow translators to work towards a better harmonization, start a glossary, or anything else, do let us know: the PrestaShop team will include a word about it in the next monthly report.
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 11]
```diff
@@ -18,11 +18,11 @@ Contributing to PrestaShop is not only about the code, it's also about taking pa
...
 
-This test was done for the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. I thus invite any latin american spanish native who wishes to help! :wink:
+This test was done for the Spanish, Mexico and Spanish, Colombia project and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. I thus invite any Latin American native who wishes to help! :wink:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
This test was done for the Spanish, Mexico and Spanish, Colombia projects, and the translation rate went respectively from 70% to 96% and 65% to 76%. :tada: To conclude this test, proofreaders of these projects are needed to vouch for the translation consistency and approve these strings. I thus invite any Latin American native who wishes to help! :wink:
```

File: news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  "Do you speak PrestaShop? – July 2020 edition"
-subtitle: "Where you'll learn that there is a way to boost translations"
+subtitle: "Where you'll learn that Thai is back on track"
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-08-07-do-you-speak-prestashop-july-2020.md]:
```suggestion
subtitle: "Where you'll learn that the Thai translation is back on track"
```

**Comment by Junebyun** [comment]
@LouiseBonnard article updated! :)

**Review by LouiseBonnard — APPROVED** [review]

**Comment by Junebyun** [comment]
Hi @ttoine, the Crowdin Monthly is ready to be published tomorrow! :)

**Pull request merged**